learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 3 Learning Outcomes
+
+## Testing
+
+- [ ] **We can test the functionality of our app**
+- [ ] We can write tests to mimic the behaviour of a user performing different actions
+- [ ] We can check that passing a given input into our tests returns the expected output
+
+### Modular functions
+
+- [ ] **We can write testable, modular functions**
+- [ ] We can write functions that add DOM nodes
+- [ ] We can write functions that remove DOM nodes
+- [ ] We can write functions that modify DOM nodes
+
+### Event listeners
+
+- [ ] **We can apply click event listeners to buttons**
+- [ ] We can pass in named functions as the second argument
+
+### Scope
+
+- [ ] **We can apply the concept of ‘scoping’ to functions and variables**
+- [ ] We can distinguish between block, function, and global scope so that we can control access to variables from different parts of our project
+
+---
+
+## Design
+
+- [ ] **We can create a clean colour palette for our project**
+- [ ] We can apply colours to HTML elements using CSS variables
+
+---
+
+## Developer Toolkit
+
+- [ ] **We can run automated tests in the browser (to check that our client side code works correctly)**
```

Comment:
This isn't really a dev tooling outcome (other than that the tests will use `console.log`)